torch_xla/csrc/runtime/runtime.cc

```diff
@@ -10,38 +10,37 @@ namespace torch_xla {
 namespace runtime {
 namespace {
 
-std::atomic<bool> g_computation_client_initialized(false);
-
-ComputationClient* CreateClient() {
-  bool was_initialized = g_computation_client_initialized.exchange(true);
-  XLA_CHECK(!was_initialized) << "ComputationClient already initialized";
+std::unique_ptr<ComputationClient> CreateClient() {
   if (sys_util::GetEnvBool("XLA_DUMP_FATAL_STACK", false)) {
     tsl::testing::InstallStacktraceHandler();
   }
 
-  ComputationClient* client;
+  std::unique_ptr<ComputationClient> client = nullptr;
 
   if (sys_util::GetEnvString(env::kEnvPjRtDevice, "") != "") {
-    client = new PjRtComputationClient();
+    client = std::make_unique<PjRtComputationClient>();
   } else {
-    g_computation_client_initialized = false;
```

Comment:
Does removing the atomic bool introduce a race condition here, or does the `std::call_once` cover that?